applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -58,4 +59,35 @@ struct le_audio_encoded_audio {
 	uint8_t num_ch;
 };
 
+int le_audio_tx_stream_stopped(uint8_t idx);
+
+int le_audio_tx_stream_started(uint8_t idx);
+
+int le_audio_tx_stream_sent(uint8_t idx);
+/**
+ * @brief	Allocates buffers and sends data to the controller
+ *
+ * @note	Should send all available channels in a single call.
+ *		I.e., this should not be called once for each channel.
+ *
+ */
+int le_audio_tx(struct bt_bap_stream *bap_streams, struct le_audio_encoded_audio enc_audio,
+		uint8_t streams_to_tx);
+
+int le_audio_tx_init(void);
+
+/**
+ * @brief	Check if an endpoint is in the given state.
+ *
+ * @note	If the endpoint is NULL, it is not in the
+ *		given state, and this function returns false.
```

Comment:
```suggestion
 * If the endpoint is NULL, it is not in the given state, and this function returns false.
```
No need for @note, in my opinion.